spec/hardware/bootdisk.fmf

```diff
@@ -0,0 +1,25 @@
+summary: |
+    Select a guest with a specific bootdisk.
+
+description: |
+    .. code-block::
+
+       # String, the bootdisk requested.
+       bootdisk: foo
+
+example:
+  - |
+    # Choose machine with given bootdisk
+    bootdisk: foo
+
+  - |
+    # Hostname matching a regular expression
+    bootdisk: "~ foo.*"
+
+  - |
+    # Hostname not matching a regular expression
+    bootdisk: "!~ foo.*"
```

Comment:
More realistic examples would be nice. For example, I never used this requirement before, therefore it would help me to understand what it might do and what it works with. Is `foo` something like `/dev/xvda`, or a vendor/product label, or something else?